aea/configurations/base.py

```diff
@@ -1867,6 +1889,10 @@ class ContractConfig(ComponentConfiguration):
     default_configuration_filename = DEFAULT_CONTRACT_CONFIG_FILE
     package_type = PackageType.CONTRACT
 
+    FIELDS_ALLOWED_TO_UPDATE = [
```

Comment:
Actually no. this should not be configurable as the interface is hard coded in the package.

Reply:
need to discuss,

going to stay as is, but as i told above, schema check does the trick, so this is presave check and provide some flexibility i'm not sure we need

Reply:
schema does check for new fields, but allows to modify already defined, so it does not cover all the cases. so this check still required